lms/static/images/icon-cert-white.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12.85 9.22">
+    <defs>
+        <style>.cls-1{fill:#fff;}</style>
+    </defs>
+    <title>cert-icon</title>
```

Comment:
I think for a11y, but it depends how the SVG is being added to the page. In this instance you are pulling it in as a background image so it won't offer any value, but it you were to embed the SVG code into the DOM I think it would add a11y context/description.